packages/auth/src/openapi/specs/id-provider.yaml

```diff
@@ -39,6 +39,16 @@ paths:
               description: Interaction id
         '401':
```

Comment:
Should this be 400?

Reply:
We also have `request_denied` in this flow possible

Reply:
> We also have `request_denied` in this flow possible

We shoud have `request_denied` in https://github.com/interledger/rafiki/pull/3094/files#diff-196f8d522cc447a08b51f0cf830f0c27ebc33e85c7cd936d96d1d6d053a1a754R46-R51

> Should this be 400?

The conditional for throwing this response can be satisfied by not finding the interaction based on the provided nonce + id, not being in the pending state, or the associated grant being revoked. I felt as though these cases sufficiently fell under "incorrect credentials".

Looking at it now, I think we could break up the conditional into 400 (or 404) for the case where it can't find the interaction because the id or the nonce don't match up, and 403 for when the grant is revoked or the interaction isn't in a pending state